src/main/resources/templates/progress/stage.html

```diff
@@ -7,11 +7,29 @@
 <div class="container">
     <div class="row">
         <div class="col-sm-12">
+
+            <form class="col-sm-6" action="#" th:action="@{/game/progress/searchTeamsOfCypher}" th:object="${search}" method="post">
```

Comment:
Předtím jsem si asi nevšiml. Ono by bylo asi lepší nemít na to separátní kontroler/akci, ale mít to filtrování zakomponované v tom stávajícícím.
Formulář pak nebude s metodou POST, ale normálně GET. Ten filtr pak bude předávaný normálně v URL.

Reply:
vytvorim na to issue ;)

Reply:
Myslíš, že ti to zaberou dlouho? Nešlo by to ještě popravit v rámci tohohle PR?

Reply:
zkusim to predelat co nejdriv - nejpozdeji zitra -> zvali jsme na svatbu tudiz nebyl moc o vikendu cas :D

Reply:
udelam to v ramci tohoto PR